staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -128,6 +132,10 @@ func setUnsetAlphaGates(known map[Feature]FeatureSpec, enabled map[Feature]bool,
 // Set, String, and Type implement pflag.Value
 var _ pflag.Value = &featureGate{}
 
+// internalPackages are packages that ignored when creating a name for featureGates These packages are in the common
+// call chains, so they'd be low entropy names for reflectors
```

Comment:
s/low entropy/unhelpful/

... this way I don't have to debate with myself any more if "low" or "high" entropy is more technically correct